static/scripts/login.js

```diff
@@ -137,6 +137,48 @@ $(document).ready(function() {
         $school.trigger('change');
     }
 
+    // EBS-System | Alert
+    function messageBuilder(message) {
```

Comment:
Ist diese Funktion die gleiche wie in der loggedIn.js? Dann bitte in eine eigene .JS Datei auslagern, bzw. alle duplizierten Teile - um duplicated code und hohen Wartungsaufwand zu vermeiden.

https://www.linkedin.com/pulse/how-code-duplication-impacts-software-maintainability-endenburg/

Reply:
Nicht gleich aber ähnlich..., wird in extra file ausgelagert